paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -51,12 +51,13 @@ inline MKLDNNMemoryFormat GetWeightsFormat(const MKLDNNMemoryFormat format,
   }
 }
 
-static mkldnn::memory::data_type GetDstType(bool is_int8,
+static mkldnn::memory::data_type GetDstType(bool is_int8, bool is_bfloat16,
```

Comment:
Since there is `mkldnn_data_type`, why not `GetDstType(std::string  mkldnn_data_type, xxx)`?

Reply:
You are right. I corrected it.

Reply:
I checked it and unfortunately, it won't work. In int8 models during warmup all conv operators are already marked as `int8` in `mkldnn_data_type` , but during warmup, computations should be run as `float32`  that is why we need to check the tensor type for int8.